app/src/utils/utils.ts

```diff
@@ -118,3 +128,31 @@ export function displayResults(
   const end = Math.min(page * perPage, numFound);
   return `${start}-${end} of ${numFound}`;
 }
+
+export const createQueryStringFromObject = (paramObj) => {
+  const params = new URLSearchParams();
+  Object.keys(paramObj).map((name, value) => {
+    params.set(
+      name.toString(),
+      name === "page" ? paramObj[name].toString() : paramObj[name]
+    );
+  });
+  return params.toString();
+};
+
+export const createCollectionsQueryStringFromObject = (paramObj) => {
+  const newParams = {};
+  Object.keys(paramObj).map((key) => {
+    if (
+      !(
+        paramObj[key] === DEFAULT_SEARCH_TERM ||
+        paramObj[key] === DEFAULT_PAGE_NUM ||
+        paramObj[key] === DEFAULT_PAGE_NUM.toString() ||
+        paramObj[key] === DEFAULT_COLLECTION_SORT
+      )
```

Comment:
You can simplify and make this more reusable if you change it to 
```
const defaultValues = [
  DEFAULT_SEARCH_TERM,
  DEFAULT_PAGE_NUM,
  DEFAULT_PAGE_NUM.toString(),
  DEFAULT_COLLECTION_SORT
];
if (!defaultValues.includes(paramObj[key])) {
  newParams[key] = paramObj[key];
}
```

Then you can reuse the function if `defaultValues` is passed as an argument for other cases.